templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -27,21 +27,6 @@ contents:
     systemCgroups: /system.slice
     nodeStatusUpdateFrequency: 10s
     nodeStatusReportFrequency: 5m
-    featureGates:
-      AlibabaPlatform: true
-      AzureWorkloadIdentity: true
-      BuildCSIVolumes: true
-      CloudDualStackNodeIPs: true
-      DisableKubeletCloudCredentialProviders: false
-      ExternalCloudProvider: true
-      ExternalCloudProviderAzure: true
-      ExternalCloudProviderGCP: true
-      ExternalCloudProviderExternal: true
-      KMSv1: true
-      NetworkLiveMigration: true
-      OpenShiftPodSecurityAdmission: true
-      PrivateHostedZoneAWS: true
-      VSphereControlPlaneMachineSet: true
```

Comment:
this removed because its replaced by another mechanism?

Reply:
yes- this PR is combined with https://issues.redhat.com/browse/MCO-817 to remove the hardcoded featuregates. They already get populated regardless